utils/users/filterAllowedUserFields.js

```diff
@@ -0,0 +1,35 @@
+const { allowedUserFieldsForUpdate } = require('~/consts/user')
+
+const isObjectEmpty = (obj) => Object.keys(obj).length === 0
```

Comment:
Just for the future, it's not necessary to use === 0 here, result will be the same without it as well